myhpom/auth.py

```diff
@@ -0,0 +1,17 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.backends import ModelBackend
+
+class EmailAuthBackend(ModelBackend):
```

Comment:
pep8 would say two spaces between imports and classes (I'm sorry for this - we should really get a linter into our CI pipeline).